app/domain/importers/content_details.rb

```diff
@@ -2,46 +2,48 @@
 
 class Importers::ContentDetails
   include Concerns::Traceable
-  attr_reader :items_service, :content_id, :base_path
+  attr_reader :items_service, :id,
 
   def self.run(*args)
     new(*args).run
   end
 
-  def initialize(content_id, base_path, _ = {})
-    @content_id = content_id
-    @base_path = base_path
+  def initialize(id, _ = {})
+    @id = id
     @items_service = ItemsService.new
   end
 
   def run
     item = nil
     begin
       time(process: :content_details) do
-        item = Dimensions::Item.find_by(content_id: content_id, latest: true)
-
-        item_raw_json = items_service.fetch_raw_json(base_path)
+        item = Dimensions::Item.find(id)
+        item_raw_json = items_service.fetch_raw_json(item.base_path)
         attributes = Metadata::Parser.parse(item_raw_json)
         content_changed = item.content_hash != attributes[:content_hash]
         item.update_attributes(attributes)
 
-        ImportQualityMetricsJob.perform_async(item.id) if content_changed
+        import_quality_metrics_for_en(item) if content_changed
```

Comment:
I would extract the logic to decide wether applying quality metrics or not to its own class. Something like:

```ruby
class Dimensions::Item
  def apply_quality_metrics?(item, new_attributes)
  end
end
```

or

```ruby
class QualityMetrics 
  def self.apply?(content_item, new_attributes)
  end
end
```